src/player.cpp

```diff
@@ -7894,7 +7894,9 @@ bool player::dispose_item( item_location &&obj, const std::string& prompt )
 void player::mend_item( item_location&& obj, bool interactive )
 {
     if( g->u.has_trait( trait_DEBUG_HS ) ) {
-        uimenu menu( true, _( "Toggle which fault?" ) );
```

Comment:
This menu is not migrated, I just changed it to use the default constructor so I can remove one of the old constructors `uimenu::uimenu( bool, const char *, ... )`, which was not working properly anyway (it assigned neither the bool flag nor the message string to the corresponding class members)